src/bundles/Header/AuthWidget.js

```diff
@@ -1,41 +1,17 @@
 import React from 'react'
-import PropTypes from 'prop-types'
+import { Provider } from 'react-redux'
 import RegisterComponent from '../../RegisterComponent'
-import { returnPath } from './helper'
 
-export const AuthWidget = (props, history) => {
+import createStore from './redux/create'
+import Header from './Header'
 
-    const _path = returnPath(history);
-
-    if (props.isAuthenticated) {
-        return (
-            <ul id="main-navigation" className="inline-links--inverted">
-                <li><a href={props.dashboardUrl}>{props.dashboardText}</a></li>
-                <li><a href={props.logoutUrl}>Sign out</a></li>
-            </ul>
-        )
-    }
-
-    return (
-    <ul id="main-navigation" className="inline-links--inverted">
-        <li><a href={props.registerUrl}>{props.registerText}</a></li>
-        <li><a href={props.loginUrl.concat(_path)}>Sign in</a></li>
-    </ul>
-    )
-}
-
-AuthWidget.propTypes = {
-    registerUrl: PropTypes.string,
-    registerText: PropTypes.string,
-    loginUrl: PropTypes.string,
-    dashboardUrl: PropTypes.string,
-    logoutUrl: PropTypes.string,
-    isAuthenticated: PropTypes.bool
-}
-
-AuthWidget.defaultProps = {
-    registerText: 'Register',
-    dashboardText: 'Dashboard'
+export const AuthWidget = (props) => {
```

Comment:
why still bother with this `AuthWidget`?

Reply:
Basically because the of the server rendered isAuthenticated prop. We would have to rebuild the python pages to not serve a server side rendered bundle, and then we would have to consolidate a store that has auth state in the front end apps. Its quite complicated :/